samples/sync/src/main/java/org/mozilla/samples/sync/MainActivity.kt

```diff
@@ -139,7 +139,7 @@ class MainActivity :
 
         // NB: ObserverRegistry takes care of unregistering this observer when appropriate, and
         // cleaning up any internal references to 'observer' and 'owner'.
-        syncManager.register(syncObserver, owner = this, autoPause = true)
+        accountManager.syncManager?.register(syncObserver, owner = this, autoPause = true)
```

Comment:
Hmmm, currently `syncManager` is only available when we have an authenticated account. However, this will be called before that! meaning that we won't actually ever register for sync events here.